src/PowerTools/source.extension.vsixmanifest

```diff
@@ -31,6 +31,9 @@ When right-clicking on a file containing a derived DbContext class, the followin
       <VisualStudio Version="12.0">
         <Edition>Pro</Edition>
       </VisualStudio>
+      <VisualStudio Version="14.0">
+35	    <Edition>Pro</Edition>
+36	  </VisualStudio>
```

Comment:
what the...? I didn't realize those got in there. Must have come over when I copied from the PR I originally submitted to codeplex. Thanks!